docs/docsite/rst/community/steering/community_steering_committee.rst

```diff
@@ -148,7 +146,7 @@ When reviewing community collection `inclusion requests <https://github.com/ansi
   * For a Non-Steering Committee review to be counted for inclusion, it MUST be checked and approved by *another* Steering Committee member.
   * Reviewers must not be involved significantly in development of the collection. They must declare any potential conflict of interest (for example, being friends/relatives/coworkers of the maintainers/authors, being users of the collection, or having contributed to that collection recently or in the past).
 
-#. After the collection gets two or more Committee member approvals, a Committee member creates a `community topic <https://github.com/ansible-community/community-topics/issues>`_ linked to the corresponding inclusion request. The issue's description says that the collection has been approved by two or more Committee members and establishes a date (a week by default) when the inclusion decision will be considered made. This time period can be used to raise concerns.
+#. After the collection gets two or more Committee member approvals, a Committee member creates a `community topic <https://forum.ansible.com/new-topic?category=project&tags=community-wg>`_ linked to the corresponding inclusion request. The issue's description says that the collection has been approved by two or more Committee members and establishes a date (a week by default) when the inclusion decision will be considered made. This time period can be used to raise concerns.
```

Comment:
This is no longer accurate, as far as I understand. Didn't we make this policy more lenient?

Reply:
It might be that we forgot to update this paragraph when we made it more lenient. But this is definitely something for a follow-up PR, since this might need a vote / more approvals, after some more history digging to find out what exactly went wrong (or not).